addons/im_livechat/static/src/embed/common/feedback_panel/transcript_sender_patch.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="im_livechat.TranscriptSender" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('o-livechat-TranscriptSender')]" position="before">
+            <div class="form-text">
+                <t t-if="state.status === STATUS.SENT">The conversation was sent.</t>
+                <t t-elif="state.status === STATUS.FAILED">An error occurred. Please try again.</t>
+                <t t-else="">Receive a copy of this conversation.</t>
+            </div>
```

Comment:
All of this should be in the base code, at least I don't see any reason not to have it.